docs/manual/system/settings.de.md

```diff
@@ -978,6 +978,16 @@ bereits erwähnten `--time-limit=1` Option wird der Prozess nach spätestens ein
 [Symfony Dokumentation](https://symfony.com/doc/current/messenger.html#consuming-messages-running-the-worker).
 {{% /notice %}}
 
+{{% notice "note" %}}
+Es kann sein, dass Mails zeitversetzt verarbeitet werden, 
+wenn der Cronjob keine Angabe für die Zeitzone hat und dann den Standard `UTC` verwendet.
+Deshalb sollte die lokale Zeitzone entweder global auf dem Server festgelegt werden oder explizit im Cronjob:
+
+```bash
+php -d date.timezone="Europe/Berlin" vendor/bin/contao-console messenger:consume --limit=10 --time-limit=1
+```
+{{% /notice %}}
```

Comment:
Not sure that is really relevant? The timezone should of course always be correctly configured in the server environment.

Reply:
Yeah, the timezones should always match. What exactly was the problem?

Reply:
well you won't get any hint and the messages in our case were processed 2 hours later because of `available_at` in the "future" - because on our server the default timezone for php cli was `UTC`

Reply:
we could also remove the example for the cronjob and only add the hint

Reply:
@fritzmg  @Toflar what do you think?

Reply:
This also applies to `contao:cron` for instance, not just `messenger:consume`. In any case, a hint that the timezone needs to be properly configured would surely not hurt, yes.

Reply:
ok I removed the example cron and just kept the hint